docs/Contributing.adoc

```diff
@@ -9,13 +9,13 @@ These are our contribution guidelines for helping out with the project. Any sugg
 * The Ansible link:https://docs.ansible.com/ansible/latest/community/code_of_conduct.html[Code of Conduct] still applies.
 * For git messages, branch names, etc., follow link:https://github.com/redhat-cop/agnosticd/blob/development/docs/git-style-guide.adoc[Git Style Guide].
 * Pull Requests should reference an *issue* whenever possible.
```

Comment:
I'm still not feeling right about this. It is kind a vague and unless it is more of a detailed requirement, no one is going to do it. 

We agree *all* PRs should have well-crafted description. Do we want to require an issue for a certain type of PR like a fix? Does the issue even matter if the reason behind the PR and what it fixes is in the description?